pkg/tests/support/helpers.py

```diff
@@ -183,209 +169,129 @@ def _default_config_path(self):
             config_path = pathlib.Path("/etc", "salt")
         return config_path
 
-    @repo_data.default
-    def _default_repo_data(self):
+    @version.default
+    def _default_version(self):
         """
-        Query to see the published Salt artifacts
-        from repo.json
+        The version to be installed at the start
         """
-        url = "https://repo.saltproject.io/salt/onedir/repo.json"
-        ret = requests.get(url)
-        data = ret.json()
-        return data
+        if not self.upgrade and not self.use_prev_version:
+            version = self.artifact_version
+        else:
+            version = self.prev_version
+            parsed = packaging.version.parse(version)
+            version = f"{parsed.major}.{parsed.minor}"
+        if self.distro_id in ("ubuntu", "debian"):
+            self.stop_services()
+        return version
 
-    def check_relenv(self, version):
+    @artifact_version.default
+    def _default_artifact_version(self):
         """
-        Detects if we are using relenv
-        onedir build
+        The version of the local salt artifacts being tested, based on regex matching
         """
-        relenv = False
-        if packaging.version.parse(version) >= packaging.version.parse("3006.0"):
-            relenv = True
-        return relenv
+        version = ""
+        for artifact in ARTIFACTS_DIR.glob("**/*.*"):
```

Comment:
```suggestion
        for artifact in ARTIFACTS_DIR.glob("**/*.*"):
```
FYI, this is the same as `ARTIFACTS_DIR.rglob("*.*")